external-builds/pytorch/build_prod_wheels.py

```diff
@@ -535,6 +548,7 @@ def do_build_pytorch(
     *,
     triton_requirement: str | None,
 ):
+    root_dir = get_rocm_path("root")
```

Comment:
What about calling this `rocm_dir`?
```suggestion
    rocm_dir = get_rocm_path("root")
```

Reply:
SG!